bundles/org.eclipse.emfcloud.modelserver.emf/src/org/eclipse/emfcloud/modelserver/emf/common/ModelController.java

```diff
@@ -119,6 +119,36 @@ public void getOne(final Context ctx, final String modeluri) {
          () -> handleError(ctx, 404, "Model '" + modeluri + "' not found!"));
    }
 
+   public void getModelElement(final Context ctx, final String modeluri, final String elementid) {
```

Comment:
Should this also match the 'getModelElementById' naming scheme? Could you please add a test case for this in ModelControllerTest.

Reply:
Yes I agree, this should be consistent :+1: